gravitee-apim-console-webui/src/management/api/policy-studio/debug/components/policy-studio-debug-response/policy-studio-debug-response.component.ts

```diff
@@ -51,6 +51,9 @@ export class PolicyStudioDebugResponseComponent implements OnChanges {
   };
 
   ngOnChanges(): void {
+    this.responseDisplayableVM = undefined;
```

Comment:
`undefined` or `null`? 🙈

Reply:
undefined like initial component state 🤷‍♂️. 

but if we had a stricter angular we should specify the initial undefined 🙈